.gitlab-ci.yml

```diff
@@ -49,13 +49,8 @@ tests:
         WORKAROUND_GHC_MMAP_CRASH: "yes"
         RUN_TESTS: "always"
 
-      - GHC_VERSION: [8.10.7, 8.8.4]
+      - GHC_VERSION: [8.10.7]
         WORKAROUND_GHC_MMAP_CRASH: "yes"
-        RUN_TESTS: "nightly"
-
-      - GHC_VERSION: 8.6.5
-        WORKAROUND_GHC_MMAP_CRASH: "yes"
-        MULTIPLE_HIDDEN: "no"
         RUN_TESTS: "always"
```

Comment:
Shouldn't we test without `multiple-hidden` on 8.10.7 here?

Reply:
You're right